modules/connectors/ninja_van/karrio/providers/ninja_van/utils.py

```diff
@@ -0,0 +1,20 @@
+
+import karrio.core as core
+
+
+class Settings(core.Settings):
+    """Ninja Van connection settings."""
+
+    # username: str  # carrier specific api credential key
```

Comment:
You need the connection settings here as well:

 ```suggestion
    client_id: str
    client_secret: str
```